src/content/docs/apis/lookup-service-api/lookup-service-api.mdx

```diff
@@ -0,0 +1,1182 @@
+
+## HTTP endpoints [#http-endpoints]
+
+### Base URL
+
+Use the base URL that's applicable for your New Relic account in your API calls.
+
+United States (US) endpoint:
+
+```
+https://nrql-lookup.service.newrelic.com
+```
+
+[European Union](/docs/using-new-relic/welcome-new-relic/get-started/introduction-eu-region-data-center) (EU) endpoint:
+
+```
+https://nrql-lookup.service.eu.newrelic.com
+```
+
+### Endpoints
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "150px" }}>
+        Method
+      </th>
+
+      <th>
+        Endpoint
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+       [`create`](#create-and-update)
+      </td>
+
+      <td>
+        `POST /v1/accounts/YOUR_ACCOUNT_ID/TABLE_NAME`
+      </td>
+
+      <td>
+        Upload a new table.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       [`update`](#create-and-update)
+      </td>
+
+      <td>
+        `PUT /v1/accounts/YOUR_ACCOUNT_ID/TABLE_NAME`
+      </td>
+
+      <td>
+        Replace an existing table.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       [`read`](#read)
+      </td>
+
+      <td>
+        `GET /v1/accounts/YOUR_ACCOUNT_ID/TABLE_NAME`
+      </td>
+
+      <td>
+        Download a table that was previously uploaded.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       [`delete`](#delete)
+      </td>
+
+      <td>
+        `DELETE /v1/accounts/YOUR_ACCOUNT_ID/TABLE_NAME`
+      </td>
+
+      <td>
+        Delete the given table.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       [`list`](#list)
+      </td>
+
+      <td>
+        `GET /v1/accounts/YOUR_ACCOUNT_ID`
+      </td>
+
+      <td>
+        List the tables previously updated for this account.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+The variables required in the above Lookups Service API endpoints are defined below. 
+
+<table>
+    <thead>
+        <tr>
+        <th style={{ width: "200px" }}>
+            Variable
+        </th>
+
+        <th style={{ width: "200px" }}>
+            Type
+        </th>
+
+        <th>
+            Description
+        </th>
+        </tr>
+    </thead>
+
+    <tbody>
+        <tr>
+        <td>
+            `YOUR_ACCOUNT_ID`
+        </td>
+
+        <td>
+            `number`
+        </td>
+
+        <td>
+            The account to which the table belongs
+        </td>
+        </tr>
+
+         <tr>
+        <td>
+            `TABLE_NAME`
+        </td>
+
+        <td>
+            `string`
+        </td>
+
+        <td>
+            A name for the stored table.  Table names must comply with the [custom event type standards](/docs/data-apis/ingest-apis/event-api/introduction-event-api/#limits):
+            * Maximum length: 255 (per the [Create custom events (Go)](/docs/apm/agents/go-agent/features/create-custom-events-go/) page)
```

Comment:
You could link to https://docs.newrelic.com/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#general for both the name & length requirements.

Reply:
Updated the link to point to https://docs.newrelic.com/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#general.